app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
@cben yea, `queue_` is the preferred convention. Not having `check` actually perform an action. (very confusing)
think out of scope for this PR.